### ISSUE TITLE
sinopia.io links should depend on env

### ISSUE THREAD

**Issue by ndushay** [LD4P/sinopia#147]
i.e.  sinopia.io has links for BFF and profile editor.  stage.sinopia.io has links to *prod* BFF and profile editor.  We'd like sinopia.io links to know "i'm stage, i need the stage links"

**Comment by michelleif** [comment]
yes please: https://development.sinopia.io menu item "Profile Editor" is going to PRD instead of to DEV as expected
